ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/spanning-tree.j2

```diff
@@ -2,14 +2,69 @@
 {% if spanning_tree is defined and spanning_tree is not none %}
 ## Spanning Tree Summary
 
-Mode: {{ spanning_tree.mode }}
+STP mode running: **{{ spanning_tree.mode }}**
 {%     if spanning_tree.mode == "mstp" %}
+{%         if spanning_tree.mst is defined and spanning_tree.mst is not none %}
+{%             if spanning_tree.mst.pvst_border is defined and spanning_tree.mst.pvst_border == true %}
+MST PSVT Border is enabled.
+{%             endif %}
+{%         endif %}
 
 ### MSTP Instance and Priority
 
-| Instance | Priority |
+| Instance(s) | Priority |
 | -------- | -------- |
-| 0 | {{ spanning_tree.priority }} |
+{%         if spanning_tree.mst_instances is defined and spanning_tree.mst_instances is not none %}
+{%            for mst_instance in spanning_tree.mst_instances | arista.avd.natural_sort %}
+| {{ mst_instance }} | {{ spanning_tree.mst_instances[mst_instance].priority }} |
```

Comment:
Missing `if` to verify if priority is set or better use ` | arista.avd.default("-")`

Reply:
Very good point, fixed it.